mitodl/open-discussions
# add user post deletion

#### What are the relevant tickets?

closes #234 

#### What's this PR do?

This adds UI so that a user can delete their own posts.

#### How should this be manually tested?

If you look at the detail view for one of your own posts you should see a little 'delete' button.

- clicking this button should open a confirmation dialog
- canceling the dialog, hitting 'escape', or clicking outside should close it
- confirming the dialog should redirect you to the channel page.

If you delete your post it's, like, _really_ deleted (forever...) so don't delete a post that is special, I guess?
